### PR TITLE
refactor(components): [slider] use JSX in Unit test

### PR DESCRIPTION
**Pull request by holazz**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by holazz** [comment]
@sxzz Please review.

File: packages/components/slider/__tests__/slider.test.tsx [hunk 1]
```diff
@@ -32,14 +32,9 @@ describe('Slider', () => {
...
     const wrapper = mount({
-      template: `
-        <slider v-model="value" :min="50"></slider>
-      `,
-      components: { Slider },
-      data() {
-        return {
-          value: 60,
-        }
+      data: () => ({ value: 60 }),
```
Comment (sxzz) [inline packages/components/slider/__tests__/slider.test.tsx:35]: Maybe we can use a ref instead of `data` property
```ts
const value = ref(60)
```

Comment (holazz) [reply packages/components/slider/__tests__/slider.test.tsx]: updated.

**Comment by holazz** [comment]
@sxzz ci fails due to https://github.com/actions/setup-node/issues/516

**Review by sxzz — APPROVED** [review]

**Pull request merged**